.github/workflows/build-push-images.yml

```diff
@@ -1,41 +0,0 @@
----
-name: Build push images
-
-"on":
-  push:
-    branches:
-      - main
-
-    paths:
-      # Only run if the images would change
-      - "Dockerfile"
-      - "rhtap/**"
-      - "copy-scripts.sh"
-
-  # ..or if manually triggered
-  workflow_dispatch:
-
-jobs:
-  build-push-images:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout source
-        uses: actions/checkout@v4
-
-      - name: Install podman
-        run: |
-          sudo apt -y install podman
-
-      - name: Podman login
-        env:
-          QUAY_USER: "${{ secrets.QUAY_PUSH_CREDS_USER }}"
-          QUAY_PASS: "${{ secrets.QUAY_PUSH_CREDS_PASS }}"
```

Comment:
I think we can remove these secrets now, they were only for use in this workflow IIRC.

Reply:
:+1:  I don't have the permissions to do that though